includes/class-newspack-newsletters.php

```diff
@@ -694,8 +705,12 @@ public static function api_permissions_check( $request ) {
 	 *
 	 * @param string  $id post ID.
 	 * @param WP_Post $post the post.
+	 * @param boolean $update Is this the initial save of the post.
```

Comment:
```suggestion
	 * @param boolean $update Is this an update of the post.
```
🤔

Reply:
Fixed in https://github.com/Automattic/newspack-newsletters/pull/106/commits/f0e592b84d3189351e952600ac581c3f69a675cf